draft-irtf-cfrg-vdaf.md

```diff
@@ -781,44 +785,43 @@ a particular encoding of that quantity as a byte string.
 
 Some common functionalities:
```

Comment:
In general I've tried to harmonize variable names across all functions:
* `x` denotes the input that's "acted upon"
* `len` denotes the length when required